spec/services/proforma_service/convert_proforma_task_to_task_spec.rb

```diff
@@ -213,11 +213,31 @@
         end
       end
 
-      context 'when file is a model-solution-placeholder (needed by proforma until issue #5 is resolved)' do
-        let(:file) { ProformaXML::TaskFile.new(id: 'ms-placeholder-file') }
+      context 'when file is used by multiple tests' do
+        let(:file) do
+          ProformaXML::TaskFile.new(
+            id: 'id',
+            content: 'very generic content',
+            filename: 'multi_referenced_file.txt',
+            used_by_grader: false,
+            visible:,
+            usage_by_lms:,
+            binary:,
+            internal_description: 'internal_description',
+            mimetype:
+          )
+        end
+
+        let(:tests) { [test, test2] }
```

Comment:
Why not inline the `test` and `test2` when they are not referenced explicitly anyway?